ports/sentry-native/vcpkg.json

```
@@ -1,6 +1,7 @@
 {
   "name": "sentry-native",
   "version-semver": "0.4.13",
+  "port-version": 1,
   "description": "Sentry SDK for C, C++ and native applications.",
   "homepage": "https://sentry.io/",
   "supports": "!(arm | (arm64 & !osx) | uwp)",
```

```suggestion
  "supports": "!uwp & (!arm & !osx)",
```

---

This is the accurate supports expression.

---

Another equivalent way to write this is:

```suggestion
  "supports": "(!uwp & !arm) | osx",
```

---

I proved that your second equivalent way is what the existing expression says in my commit message (by translating the original into DNF). I was trying to avoid making changes when I could show that there was no disagreement between the 2 places, but I can make this change.